File: examples/museum/oseg_generated/ruby/UpdateSpecialEvent_default_example.rb
require "openapimuseum_client"

OpenapiMuseumClient.configure do |config|
end

special_event_fields = OpenapiMuseumClient::SpecialEventFields.new
special_event_fields.name = nil
special_event_fields.location = "On the beach."
special_event_fields.event_description = nil
special_event_fields.price = 15
special_event_fields.dates = nil

begin
    api_caller = OpenapiMuseumClient::EventsApi.new

    response = api_caller.update_special_event(
        "dad4bce8-f5cb-4078-a211-995864315e39",
        special_event_fields,
    )

    p response
rescue OpenapiMuseumClient::ApiError => e
    puts "Exception when calling Events#update_special_event: #{e}"
end

File: examples/museum/oseg_generated/python/UpdateSpecialEvent_default_example.py
from datetime import date, datetime
from pprint import pprint

from openapimuseum_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration()

with ApiClient(configuration) as api_client:
    special_event_fields = models.SpecialEventFields(
        name=None,
        location="On the beach.",
        event_description=None,
        price=15,
        dates=None,
    )

    try:
        response = api.EventsApi(api_client).update_special_event(
            event_id="dad4bce8-f5cb-4078-a211-995864315e39",
            special_event_fields=special_event_fields,
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling Events#update_special_event: %s\n" % e)
